packages/steps-s3-copy/src/lib/copy-map-construct.ts

```diff
@@ -147,17 +186,36 @@ export class CopyMapConstruct extends Construct {
       }),
     });
 
-    this.distributedMap = new S3JsonlDistributedMap(this, id + "CopyMap", {
+    this.distributedMap = new S3JsonlDistributedMap(this, id, {
       toleratedFailurePercentage: 0,
       maxItemsPerBatch: props.maxItemsPerBatch,
       maxConcurrency: props.maxConcurrency,
-      batchInput: undefined,
+      batchInput: {
+        glacierFlexibleRetrievalThawDays: 1,
+        glacierFlexibleRetrievalThawSpeed: props.aggressiveTimes
+          ? "Expedited"
+          : "Bulk",
+        glacierDeepArchiveThawDays: 1,
+        glacierDeepArchiveThawSpeed: props.aggressiveTimes
+          ? "Expedited"
+          : "Bulk",
+        intelligentTieringArchiveThawDays: 1,
+        intelligentTieringArchiveThawSpeed: props.aggressiveTimes
+          ? "Standard"
+          : "Bulk",
+        intelligentTieringDeepArchiveThawDays: 1,
+        intelligentTieringDeepArchiveThawSpeed: props.aggressiveTimes
+          ? "Standard"
+          : "Bulk",
+      },
```

Comment:
I will put in a new issue - so not for this PR - but we will eventually want these parameters to be allowed to be specified by the invoker of the copy (so possibly bringing this entire data structure up to the Steps invoke parameter level)